python-lib/ontology_tagging_formatting.py

```diff
@@ -38,18 +38,32 @@ def _arrange_columns_order(
             df.reset_index(inplace=True)
         return df
 
+    def _apply_matcher(
```

Comment:
It's hard to understand that language_column is sometimes ignored. I would suggest to (i) add a default value in the function and (ii) explain it in the docstring

Reply:
done in  ux/improvments  :  0081f34